packages/api-logs/index.jsx

```diff
@@ -222,8 +222,36 @@ class Logs extends React.Component {
   }
 
   render() {
-    const { query, baseUrl, group } = this.props;
-    if (!group) return null;
+    const { query, baseUrl, group, loginUrl } = this.props;
+    if (!group)
+      return (
+        <div className="logs">
+          <div>
+            <h3>Logs</h3>
+          </div>
+          <div>
+            <div className="logs-login">
+              <svg fill="none" height="19" viewBox="0 0 24 19" width="24" xmlns="http://www.w3.org/2000/svg">
+                <path d="M3.29102 8.82048C3.29102 8.43813 3.59388 8.12817 3.96749 8.12817H12.0851C12.4587 8.12817 12.7616 8.43813 12.7616 8.82048C12.7616 9.20283 12.4587 9.51279 12.0851 9.51279H3.96749C3.59388 9.51279 3.29102 9.20283 3.29102 8.82048Z" />
+                <path d="M3.29102 3.5751C3.29102 3.16916 3.61256 2.84009 4.00921 2.84009C4.40587 2.84009 4.72741 3.16916 4.72741 3.5751C4.72741 3.98104 4.40587 4.31011 4.00921 4.31011C3.61256 4.31011 3.29102 3.98104 3.29102 3.5751Z" />
+                <path d="M5.99707 3.5751C5.99707 3.16916 6.31862 2.84009 6.71527 2.84009C7.11192 2.84009 7.43347 3.16916 7.43347 3.5751C7.43347 3.98104 7.11192 4.31011 6.71527 4.31011C6.31862 4.31011 5.99707 3.98104 5.99707 3.5751Z" />
+                <path d="M8.70361 3.5751C8.70361 3.16916 9.02516 2.84009 9.42181 2.84009C9.81846 2.84009 10.14 3.16916 10.14 3.5751C10.14 3.98104 9.81846 4.31011 9.42181 4.31011C9.02516 4.31011 8.70361 3.98104 8.70361 3.5751Z" />
+                <path d="M3.29102 11.5893C3.29102 11.2069 3.59388 10.897 3.96749 10.897H14.791C15.1646 10.897 15.4675 11.2069 15.4675 11.5893C15.4675 11.9716 15.1646 12.2816 14.791 12.2816H3.96749C3.59388 12.2816 3.29102 11.9716 3.29102 11.5893Z" />
+                <path d="M3.29102 14.3586C3.29102 13.9762 3.59388 13.6663 3.96749 13.6663H9.37925C9.75286 13.6663 10.0557 13.9762 10.0557 14.3586C10.0557 14.7409 9.75286 15.0509 9.37925 15.0509H3.96749C3.59388 15.0509 3.29102 14.7409 3.29102 14.3586Z" />
+                <path
+                  clipRule="evenodd"
+                  d="M21.9895 1.70441H2.18145V16.4377H21.9895V1.70441ZM2.18145 0.0710449C1.3 0.0710449 0.585449 0.802327 0.585449 1.70441V16.4377C0.585449 17.3398 1.3 18.071 2.18145 18.071H21.9895C22.8709 18.071 23.5854 17.3398 23.5854 16.4377V1.70441C23.5854 0.802327 22.8709 0.0710449 21.9895 0.0710449H2.18145Z"
+                  fillRule="evenodd"
+                />
+              </svg>
+              <p>Log in to view your API logs for this endpoint</p>
+              <a className="logs-login-button" href={loginUrl}>
```

Comment:
```suggestion
              <a className="logs-login-button" href={loginUrl} rel="noreferrer noopener">
```

Reply:
Do we need this since it's opening it in the same tab? I figured it'd be clunky to add `target="_blank"`, etc. since logging in would redirect back to the documentation.

Reply:
Good point! It's only needed if we're opening in a new tab.

Reply:
Also tested this on https://mathiasbynens.github.io/rel-noopener/ just to make sure, and it checks out. We don't need it here.